[Security Solution][Exceptions] - Updates autocomplete validation for numbers and boolean

yctercero opened the pull request:
## Summary

This PR chips ⛏️  a bit at some stricter value validations that have been discussed. Further validation is needed, but this adds some more basic validation.

- **Current:** if selected field is of type `boolean` users can add custom values in combo box
  - **Now:** if selected field is of type `boolean` users can only select `true` or `false`
- **Current:** if selected field is of type `number` (that is kibana type number) user can input any values
  - **Now:** if selected field is of type `number` and no autocomplete suggestions are available, number input is used to restrict users
- **Current:** for operator `match_any` it's conducting an autocomplete search after each selection resulting in some jumpy/weird behavior
  - **Now:** only conducts autocomplete search on initial field selection and if user enters value to search
- **Current:** only validations on type date
  - **Now:** validation on type (Kibana type) date, number
- **Current:** input would show red when there was an error but user could still submit
  - **Now:** submit button is disabled if error exists

### TODO
- [ ] can continue improving validation, this was just adding some basics

#### Boolean
![boolean](https://user-images.githubusercontent.com/10927944/90676181-989a3180-e229-11ea-9f16-e3ab9ce3f8c7.gif)

#### Number w/ autocomplete values available
![number_values_available](https://user-images.githubusercontent.com/10927944/89556893-05e8a400-d7e0-11ea-97b4-f30df32d45d4.gif)

#### Number w/o autocomplete values available
![number_no_values](https://user-images.githubusercontent.com/10927944/89557066-40ead780-d7e0-11ea-993d-84a14f2c9706.gif)

#### Match any jitter
##### Before
![match_any_jitter](https://user-images.githubusercontent.com/10927944/89944995-af6bd300-dbee-11ea-854c-c3e184ab1e5d.gif)

##### After
![match_any_jitter_fix](https://user-images.githubusercontent.com/10927944/89945087-cca0a180-dbee-11ea-991f-136cefaedfc3.gif)

##### Validation (disabling submit)
![validation](https://user-images.githubusercontent.com/10927944/90676217-a9e33e00-e229-11ea-8f37-c665fb56c39f.gif)

### Checklist

Delete any items that are not applicable to this PR.

- [x] Any text added follows [EUI's writing guidelines](https://elastic.github.io/eui/#/guidelines/writing), uses sentence case text and includes [i18n support](https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/README.md)
- [x] [Unit or functional tests](https://github.com/elastic/kibana/blob/master/CONTRIBUTING.md#cross-browser-compatibility) were updated or added to match the most common scenarios
- [ ] This was checked for [keyboard-only and screenreader accessibility](https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Cross_browser_testing/Accessibility#Accessibility_testing_checklist)
- [ ] This renders correctly on smaller devices using a responsive layout. (You can test this [in your browser](https://www.browserstack.com/guide/responsive-testing-on-local-server)
- [ ] This was checked for [cross-browser compatibility](https://www.elastic.co/support/matrix#matrix_browsers)

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/66980/)
* Commit: 4133023c712be9f53f492bc958200f755da34483
* This comment will update when the build is complete

### Failed CI Steps
* [Check Types](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/66980/execution/node/370/log)

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/66980/","number":"66980","commit":"4133023c712be9f53f492bc958200f755da34483"}]}
PIPELINE-->

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/67522/)
* Commit: c832711a84c6f1da64c97399dde913f8521f0d08
* This comment will update when the build is complete

### Failed CI Steps
* [Check Types](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/67522/execution/node/376/log)

### History
* :broken_heart: [Build #66980](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/66980/) failed 4133023c712be9f53f492bc958200f755da34483

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/66980/","number":"66980","commit":"4133023c712be9f53f492bc958200f755da34483"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/67522/","number":"67522","commit":"c832711a84c6f1da64c97399dde913f8521f0d08"}]}
PIPELINE-->

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/67573/)
* Commit: 8798b4d5969ef08c4e9d23aeec209d378332466e

### Build metrics
#### async chunks size
| id | [value](https://github.com/elastic/kibana/commit/8798b4d5969ef08c4e9d23aeec209d378332466e) | diff | [baseline](https://github.com/elastic/kibana/commit/f621b0e2b68cd47341074a93c96ab7a5ede3d9bc) |
| --- | --- | --- | --- |
| securitySolution | 7.3MB | +5.5KB | 7.3MB |

### History
* :broken_heart: [Build #67522](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/67522/) failed c832711a84c6f1da64c97399dde913f8521f0d08
* :broken_heart: [Build #66980](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/66980/) failed 4133023c712be9f53f492bc958200f755da34483

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/66980/","number":"66980","commit":"4133023c712be9f53f492bc958200f755da34483"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/67522/","number":"67522","commit":"c832711a84c6f1da64c97399dde913f8521f0d08"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/67573/","number":"67573","commit":"8798b4d5969ef08c4e9d23aeec209d378332466e"}]}
PIPELINE-->

elasticmachine commented:
Pinging @elastic/siem (Team:SIEM)

yctercero commented:
@elasticmachine merge upstream

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/67974/)
* Commit: a4426472f937bc2602c43a32301fbfd61613bc93

### Build metrics
#### async chunks size
| id | [value](https://github.com/elastic/kibana/commit/a4426472f937bc2602c43a32301fbfd61613bc93) | diff | [baseline](https://github.com/elastic/kibana/commit/4c810be33595d478c6eb802170d0644b32e62898) |
| --- | --- | --- | --- |
| securitySolution | 7.3MB | +5.5KB | 7.3MB |

### History
* :green_heart: [Build #67573](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/67573/) succeeded 8798b4d5969ef08c4e9d23aeec209d378332466e
* :broken_heart: [Build #67522](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/67522/) failed c832711a84c6f1da64c97399dde913f8521f0d08
* :broken_heart: [Build #66980](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/66980/) failed 4133023c712be9f53f492bc958200f755da34483

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/66980/","number":"66980","commit":"4133023c712be9f53f492bc958200f755da34483"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/67522/","number":"67522","commit":"c832711a84c6f1da64c97399dde913f8521f0d08"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/67573/","number":"67573","commit":"8798b4d5969ef08c4e9d23aeec209d378332466e"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/67974/","number":"67974","commit":"a4426472f937bc2602c43a32301fbfd61613bc93"}]}
PIPELINE-->

spong commented:
Looks like when inputting an invalid value after a valid value has been selected the valid value persists and is what is submitted. Perhaps this invalid input should clear the field's value?

Empty -> Valid -> Invalid
<p align="center">
  <img width="500" src="https://user-images.githubusercontent.com/2946766/90297051-8e9fb980-de4a-11ea-9425-4f0374e29477.gif" />
</p>

Empty -> Invalid
<p align="center">
  <img width="500" src="https://user-images.githubusercontent.com/2946766/90297152-d292be80-de4a-11ea-877c-394a1c7f1caa.png" />
</p>

spong commented:
Just a bounds nit (so no need to fix here), but seems the field is truncated and displayed in scientific notation and the number picker no longer functions. This doesn't appear to cause any issues though, as modifying & re-saving the exception keeps the original value.

<p align="center">
  <img width="500" src="https://user-images.githubusercontent.com/2946766/90297512-f99dc000-de4b-11ea-8fd0-92e4547810f3.gif" />
</p>

Also note, number validation doesn't take effect for `is one of` or `is not one of` operators. I don't think we can get the number picker to work with the multi-select, but looks like we can at least limit to numbers as per the [ComboBox EUI docs](https://elastic.github.io/eui/#/forms/combo-box) (_Custom options only, with validation_).
<p align="center">
  <img width="500" src="https://user-images.githubusercontent.com/2946766/90297634-687b1900-de4c-11ea-9ce8-d43c160d7857.png" />
</p>

spong commented on x-pack/plugins/security_solution/public/common/components/autocomplete/field_value_match.tsx:
```diff
@@ -98,27 +105,72 @@ export const AutocompleteFieldMatchComponent: React.FC<AutocompleteFieldMatchPro
+        isInvalid={!isValid}
+        onFocus={setIsTouchedValue}
+        sortMatchesBy="startsWith"
+        data-test-subj="valuesAutocompleteComboBox matchComboxBox"
```
There's a few more of these floating around outside this change-set, so just a heads up. I'm seeing 8 in total when searching `matchComboxBox` in my ide.

```suggestion
        data-test-subj="valuesAutocompleteComboBox matchComboBox"
```

yctercero replied:
Updated!

spong commented on x-pack/plugins/security_solution/public/common/components/autocomplete/field_value_match.tsx:
```diff
@@ -98,27 +105,72 @@ export const AutocompleteFieldMatchComponent: React.FC<AutocompleteFieldMatchPro
-      async
-    />
-  );
+  const getDefaultInput = (): JSX.Element => {
```
I'm a little rusty on the component render optimization front, but I believe you can memoize this function, and wrap the above `handler` functions in a `useCallback` to maintain references and minimize some unnecessary renders. 

I'm seeing ~5 renders per entry when opening the exceptions modal/creating a new entry, but it may just be the parents props causing this. The UI still  _feels snappy_, and that's always a good litmus test, but just wanted to give a heads up so no changes necessary here. There will be plenty of fun stuff to uncover when you go on your perf adventures 🙂

yctercero replied:
Updated!

spong commented on x-pack/plugins/security_solution/public/common/components/autocomplete/field_value_match_any.tsx:
```diff
@@ -64,14 +67,8 @@ export const AutocompleteFieldMatchAnyComponent: React.FC<AutocompleteFieldMatch
-        patterns: indexPattern,
-      });
-    }
+  const handleSearchChange = (searchVal: string) => {
```
As above, can leverage a `useCallback` for these handlers to maintain references as passed into the below `EuiComboBox` component.

spong commented on x-pack/plugins/security_solution/public/common/components/autocomplete/hooks/use_field_value_autocomplete.test.ts:
```diff
@@ -113,18 +116,71 @@ describe('useFieldValueAutocomplete', () => {
+          query: '',
+        })
+      );
+      // Note: initial `waitForNextUpdate` is hook initialization
```
Thanks for the comment 👍

yctercero commented:
@spong Thanks so much for the 👀 . I went back and fixed the various things you noted. I made the input for boolean types into a select so now user can't even type anything in there. I also added validation for numbers when operator is `one of`. Lastly, updated to show errors below inputs and disable submit button if any errors exist. Let me know what you think!

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/69177/)
* Commit: d31743f6b210019ecbd22c1ef0650575225a8705
* This comment will update when the build is complete

### Failed CI Steps
* [Check Types](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/69177/execution/node/364/log)

### History
* :green_heart: [Build #67974](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/67974/) succeeded a4426472f937bc2602c43a32301fbfd61613bc93
* :green_heart: [Build #67573](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/67573/) succeeded 8798b4d5969ef08c4e9d23aeec209d378332466e
* :broken_heart: [Build #67522](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/67522/) failed c832711a84c6f1da64c97399dde913f8521f0d08
* :broken_heart: [Build #66980](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/66980/) failed 4133023c712be9f53f492bc958200f755da34483

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/66980/","number":"66980","commit":"4133023c712be9f53f492bc958200f755da34483"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/67522/","number":"67522","commit":"c832711a84c6f1da64c97399dde913f8521f0d08"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/67573/","number":"67573","commit":"8798b4d5969ef08c4e9d23aeec209d378332466e"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/67974/","number":"67974","commit":"a4426472f937bc2602c43a32301fbfd61613bc93"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/69177/","number":"69177","commit":"d31743f6b210019ecbd22c1ef0650575225a8705"}]}
PIPELINE-->

spong commented on x-pack/plugins/security_solution/public/common/components/autocomplete/field_value_match.tsx:
```diff
@@ -38,24 +47,45 @@ export const AutocompleteFieldMatchComponent: React.FC<AutocompleteFieldMatchPro
+  const [searchQuery, setSearchQuery] = useState('');
   const [touched, setIsTouched] = useState(false);
-  const [isLoadingSuggestions, suggestions, updateSuggestions] = useFieldValueAutocomplete({
+  const [error, setError] = useState<string | null | undefined>(undefined);
```
Is `null` necessary here? Looks like all calls to `handleError` provide either `undefined` or the results of `paramIsValid()` which is typed to return `string | undefined `.

```suggestion
  const [error, setError] = useState<string | undefined>(undefined);
```

spong commented on x-pack/plugins/security_solution/public/common/components/autocomplete/field_value_match_any.tsx:
```diff
@@ -36,12 +39,16 @@ export const AutocompleteFieldMatchAnyComponent: React.FC<AutocompleteFieldMatch
+  const [searchQuery, setSearchQuery] = useState('');
   const [touched, setIsTouched] = useState(false);
-  const [isLoadingSuggestions, suggestions, updateSuggestions] = useFieldValueAutocomplete({
+  const [error, setError] = useState<string | null | undefined>(undefined);
```
As above:

> Is `null` necessary here? Looks like all calls to `handleError` provide either `undefined` or the results of `paramIsValid()` which is typed to return `string | undefined `.

```suggestion
  const [error, setError] = useState<string | undefined>(undefined);
```

spong commented:
Seeing an issue where switching fields (`string`->`boolean`) when validation is failed results in newly selected values from validatin:

<p align="center">
  <img width="500" src="https://user-images.githubusercontent.com/2946766/90689119-c2088c80-e22c-11ea-8e2d-d369989ef36d.gif" />
</p>

spong commented:
We're in _super-nit_ territory at this point so feel free to ignore.... 😅 

When changing focus the validation error will bounce between `empty` and `type` errors. I suppose my expectation would be for it stay `not a valid number`, but understand that the value isn't set (to fix that other issue above :), so definitely not a big deal. There's always UI/UX cornercases floating around validation, but you've covered all the main cases and gotchas, so we're already in a really good spot with all the improvements you've made in this PR -- many thanks @yctercero!! 🙂 
<p align="center">
  <img width="500" src="https://user-images.githubusercontent.com/2946766/90698068-1582d680-e23d-11ea-8b58-a9770e39ef0d.gif" />
</p>


I could see this going either way, but when a validation error is displayed it increases the height of the exception entry and will move the trash icon down to keep it vertically centered. May want to keep vertically centered with the value input field 
(unless this is necessary to support smaller viewports).

<p align="center">
  <img width="500" src="https://user-images.githubusercontent.com/2946766/90698085-20d60200-e23d-11ea-98a5-7471ec4ca02a.png" />
</p>

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/80354/)
* Commit: 52d42790b2242306fd0495679792d9edb9d1d51b
* This comment will update when the build is complete

### Failed CI Steps
* [Check Types](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/80354/execution/node/364/log)

### History
* :broken_heart: [Build #69177](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/69177/) failed d31743f6b210019ecbd22c1ef0650575225a8705
* :green_heart: [Build #67974](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/67974/) succeeded a4426472f937bc2602c43a32301fbfd61613bc93
* :green_heart: [Build #67573](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/67573/) succeeded 8798b4d5969ef08c4e9d23aeec209d378332466e
* :broken_heart: [Build #67522](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/67522/) failed c832711a84c6f1da64c97399dde913f8521f0d08
* :broken_heart: [Build #66980](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/66980/) failed 4133023c712be9f53f492bc958200f755da34483

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/66980/","number":"66980","commit":"4133023c712be9f53f492bc958200f755da34483"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/67522/","number":"67522","commit":"c832711a84c6f1da64c97399dde913f8521f0d08"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/67573/","number":"67573","commit":"8798b4d5969ef08c4e9d23aeec209d378332466e"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/67974/","number":"67974","commit":"a4426472f937bc2602c43a32301fbfd61613bc93"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/69177/","number":"69177","commit":"d31743f6b210019ecbd22c1ef0650575225a8705"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/80354/","number":"80354","commit":"52d42790b2242306fd0495679792d9edb9d1d51b"}]}
PIPELINE-->

dplumlee commented on x-pack/plugins/security_solution/public/common/components/exceptions/edit_exception_modal/index.tsx:
```diff
@@ -110,8 +110,11 @@ export const EditExceptionModal = memo(function EditExceptionModal({
   >([]);
   const { addError, addSuccess } = useAppToasts();
   const { loading: isSignalIndexLoading, signalIndexName } = useSignalIndex();
+  const memoSignalIndexName = useMemo(() => (signalIndexName !== null ? [signalIndexName] : []), [
```
this is working for me, good find 👍

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/80368/)
* Commit: a413d9a3c92c37eb1ab2d5a9a265307a3b7e1404

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### async chunks size
| id | [before](https://github.com/elastic/kibana/commit/a413d9a3c92c37eb1ab2d5a9a265307a3b7e1404) | [after](https://github.com/elastic/kibana/commit/7031ea4f7b61c1dde96232cdb8ca52cac3fc9934) | diff |
| --- | --- | --- | --- |
| `securitySolution` | 10.5MB | 10.5MB | +20.4KB |

### History
* :broken_heart: [Build #80354](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/80354/) failed 52d42790b2242306fd0495679792d9edb9d1d51b
* :broken_heart: [Build #69177](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/69177/) failed d31743f6b210019ecbd22c1ef0650575225a8705
* :green_heart: [Build #67974](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/67974/) succeeded a4426472f937bc2602c43a32301fbfd61613bc93
* :green_heart: [Build #67573](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/67573/) succeeded 8798b4d5969ef08c4e9d23aeec209d378332466e
* :broken_heart: [Build #67522](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/67522/) failed c832711a84c6f1da64c97399dde913f8521f0d08

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/67522/","number":"67522","commit":"c832711a84c6f1da64c97399dde913f8521f0d08"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/67573/","number":"67573","commit":"8798b4d5969ef08c4e9d23aeec209d378332466e"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/67974/","number":"67974","commit":"a4426472f937bc2602c43a32301fbfd61613bc93"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/69177/","number":"69177","commit":"d31743f6b210019ecbd22c1ef0650575225a8705"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/80354/","number":"80354","commit":"52d42790b2242306fd0495679792d9edb9d1d51b"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/80368/","number":"80368","commit":"a413d9a3c92c37eb1ab2d5a9a265307a3b7e1404"}]}
PIPELINE-->

elasticmachine commented:
Pinging @elastic/security-solution (Team: SecuritySolution)
